ignite/cmd/chain.go

```diff
@@ -164,6 +184,28 @@ func toolsMigrationPreRunHandler(cmd *cobra.Command, session *cliui.Session) (er
 	return os.WriteFile(toolsFilename, buf.Bytes(), 0o644)
 }
 
+func bufMigrationPreRunHandler(cmd *cobra.Command, session *cliui.Session) error {
+	appPath := flagGetPath(cmd)
+	hasFiles := chain.CheckBufFiles(appPath)
+	if hasFiles {
+		return nil
+	}
+	if err := session.AskConfirm(msgMigrationBuf); err != nil {
+		return fmt.Errorf("build the protobuf files with the protoc binary is only supported on version " +
+			"`v0.26.1` or below. Please downgrade your Ignite version")
```

Comment:
This could be a custom error:

```go
var ErrProtocUnsupported = errors.New("code generation using protoc is only supported by Ignite CLI v0.26.1 or older")
```